build/Targets/Versions.props

```diff
@@ -49,6 +49,8 @@
     <NuGetPackageRoot Condition="'$(NuGetPackageRoot)' == '' AND '$(OS)' == 'Windows_NT'">$(UserProfile)/.nuget/packages/</NuGetPackageRoot>
     <NuGetPackageRoot Condition="'$(NuGetPackageRoot)' == '' AND '$(OS)' != 'Windows_NT'">$(HOME)/.nuget/packages/</NuGetPackageRoot>
     <NuGetPackagesDirectory>$(NuGetPackageRoot)</NuGetPackagesDirectory>
+
+    <Version Condition="$(Version) == ''">$(NuGetPerBuildPreReleaseVersion)</Version>
```

Comment:
The downgrade warnings are bug NuGet/Home#4620.

> @nguerrera is there any way to generate a proper runtimeconfig.json and deps.json without referencing Microsoft.NETCore.App?

I just did a quick test I am able to do this without referencing Microsoft.NETCore.App. Can you file a bug with repro steps on dotnet/sdk?